app/js/ui.js

```diff
@@ -1,22 +1,21 @@
+//Get csv from user upload
 document.getElementById('upload_link').onclick = function() {
     document.getElementById('input_csv').click();
     userUploadFlag = true;
 };
 
-/*
- * Code to run when document is ready
- */
+//Modal settings, as defined by Material Design Guidelines
 $(document).ready(function() {
     // the "href" attribute of the modal trigger must specify the modal ID that wantsupload_link_topo to be triggered
     $('.modal').modal();
 
     $('.modal').modal({
-        dismissible: true, // Modal can be dismissed by clicking outside of the modal
-        opacity: .5, // Opacity of modal background
-        inDuration: 300, // Transition in duration
-        outDuration: 200, // Transition out duration
-        startingTop: '4%', // Starting top style attribute
-        endingTop: '10%', // Ending top style attribute
+        dismissible: true,  // Modal can be dismissed by clicking outside of the modal
+        opacity: .5,        // Opacity of modal background
+        inDuration: 300,    // Transition in duration
```

Comment:
I don't have a reference for this, but I am pretty sure Google Javascript style guidelines requires comments to be dedicated to their own line.